lib/fine-types/test/data/Cardano/Ledger/Babbage/Block.fine

```diff
@@ -0,0 +1,48 @@
+module Block where
+
+import Crypto(Proof,Seed,Sig,VKey,VKey_ev);
+import PParams(KESPeriod,ProtVer);
+import Address(Slot);
+import Tx(Tx);
+
+{-----------------------------------------------------------------------------
+    Block Definitions
+    Shelley spec, Figure 53
+------------------------------------------------------------------------------}
+
+-- Abstract types
+
+HashHeader    = _; --^ hash of a block header
+HashBBody     = _; --^ hash of a block body
+BlockNo       = ℕ; --^ block number
+
+--| Operational Certificate
+OCert         =
+    { vk_hot  : VKey_ev --^ operational (hot) key
+    , n       : ℕ --^ certificate issue number
+    , c_0     : KESPeriod --^ start KES period
+    , σ       : Sig --^ cold key signature
+    };
+
+--| Block Header Body
+BHBody        =
+    { prev    : HashHeader? --^ hash of previous block header
+    , vk      : VKey --^ block issuer
+    , vrfVk   : VKey  --^ VRF verification key
+    , blockno : BlockNo --^ block number
+    , slot    : Slot --^ block slot
+    , vrfRes  : Seed --^ VRF result value
+    , η       : Seed --^ nonce
+    , prf_η   : Proof --^ nonce proof
+    , ℓ       : {x : ℚ | x ∈ [0, 1]}  --^ leader election value
+    , prf_ℓ   : Proof --^ leader election proof
```

Comment:
```suggestion
    , prf     : Proof --^ vrf proof
```

They really removed fields from the Block in Babbage. 🤓 If you ask me, they should have highlighted `prf` in yellow as well. 🤔 Probably difficult to highlight things that have been removed, though.

Reply:
Ohh, thanks for really looking into